roles/test_operator/defaults/main.yml

```diff
@@ -68,6 +68,9 @@ cifmw_test_operator_tobiko_image: "{{ cifmw_test_operator_tobiko_registry }}/{{
 cifmw_test_operator_tobiko_image_tag: current-podified
 cifmw_test_operator_tobiko_testenv: scenario
 cifmw_test_operator_tobiko_version: master
+cifmw_test_operator_tobiko_pytest_addopts: null
+cifmw_test_operator_tobiko_prevent_create: null
+cifmw_test_operator_tobiko_num_processes: null
```

Comment:
According to docs this should be auto

Reply:
I will explain better in the README file that the default values of these new params are `null`, that this means corresponding env variables are not set in the tobiko pod and that the tobiko fw applies the corresponding default values in that case.